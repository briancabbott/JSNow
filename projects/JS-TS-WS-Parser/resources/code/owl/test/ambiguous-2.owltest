---program = stmt*
stmt =
 expr : expr
 'if' expr 'then' expr : if
expr =
 identifier : ident
 identifier '(' identifier ')' : function
 '(' identifier ')' : parens
