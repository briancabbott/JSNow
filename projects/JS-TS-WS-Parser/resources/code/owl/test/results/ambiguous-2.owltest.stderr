error: this grammar is ambiguous

. a ( b ) 

  can be parsed in two different ways: as

. a ( b )       
  expr:function 
  stmt:expr---- 
  program------ 

  or as

. a          ( b )       
  expr:ident expr:parens 
  stmt:expr- stmt:expr-- 
  program--------------- 

